### PR TITLE
Fixes incorrect markdown rendering

### PR DESCRIPTION
**Pull request by javierarce**
This PR fixes a problem with the markdown rendering of the map descriptions.

Original issue: #4811.

### CODE REVIEW

**Comment by javierarce** [comment]
Please, have a look, @matallo.

File: app/views/admin/visualizations/public_map.html.erb [hunk 1]
```diff
@@ -127,7 +127,7 @@
             <div class="js-user-meta">
               <h1 class="PublicMap-title js-PublicMap-title Title Title--m"><%= @visualization.name %></h1>
               <% if @visualization.description_clean %>
-                <p class="PublicMap-description"><%= @visualization.description %></p>
+                <div class="PublicMap-description"><%= raw @visualization.description_html_safe %></div>
```
Comment (javierarce) [inline app/views/admin/visualizations/public_map.html.erb]: The change to `div` is because the Recarpet gem surrounds the output between `p` tags.

Comment (matallo) [inline app/views/admin/visualizations/public_map.html.erb]: yeah, I supposed so, testing right now :)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5513/))

**Comment by matallo** [comment]
uhm, maybe I'm missing something?

screen shot 2015-07-31 at 12 54 14
screen shot 2015-07-31 at 12 54 24

user
https://cartodb-staging.com/superadmin/users/2489

**Comment by javierarce** [comment]
@matallo: I tried with `[test](http://www.cartodb.com)` and it worked ¿?

**Comment by matallo** [comment]
oh! :green_heart: then

**Comment by javierarce** [comment]
Super, thanks for the review!

**Pull request merged**